test/system/170-run-userns.bats

```diff
@@ -52,3 +52,14 @@ function _require_crun() {
     run_podman 125 run --group-add keep-groups --group-add 457 $IMAGE id
     is "$output" ".*the '--group-add keep-groups' option is not allowed with any other --group-add options" "Check group leaked into container"
 }
+
+@test "podman userns=auto in config file" {
+    skip_if_rootless "the user might not have enough IDs"
```

Comment:
Is there a way you could compute this from the current context's `/proc/self/uid_map` and `/etc/subuid`?

Reply:
we could parse `/etc/subuid`, dropped this line for an explicit check.